test/date.js

```diff
@@ -153,15 +153,25 @@ describe('Date Component', () => {
           'date-field': error
         };
         date.hooks['pre-render'](req, res, next);
-        expect(res.render).to.have.been.calledWith(path.resolve(__dirname, '../templates/date.html'), { error, key: 'date-field', legend: null, hint: null });
+        expect(res.render).to.have.been.calledWith(path.resolve(__dirname, '../templates/date.html'), { error, key: 'date-field', legend: null, legendClassName: '', hint: null });
       });
 
-      it('calls res.render with template, key, legend and hint with values NOT null', () => {
+      it('calls res.render with template, key, legend and hint with values NOT null OR empy string', () => {
         req.translate = sinon.stub();
         req.translate.withArgs('fields.date-field.legend').returns('you legend');
         req.translate.withArgs('fields.date-field.hint').returns('some hint');
         date.hooks['pre-render'](req, res, next);
-        expect(res.render).to.have.been.calledWith(path.resolve(__dirname, '../templates/date.html'), { error: undefined, key: 'date-field', legend: 'you legend', hint: 'some hint' });
+        expect(res.render).to.have.been.calledWith(path.resolve(__dirname, '../templates/date.html'), { error: undefined, key: 'date-field', legendClassName: '', legend: 'you legend', hint: 'some hint' });
```

Comment:
I totally agree and I actually only wanted to test those elements.  I think my issues are
1. I don't understand a lot of the code-base
2. I kind of know what I'm looking to do but then I don't know how to implement it.  
3. I'm very raw in my skillset which is kind of relevant to number 2.  I'm not always sure what to google and what are to look in